humilis/environment.py

```diff
@@ -50,7 +50,8 @@ def __init__(self, yml_path, logger=None, stage=None, vault_layer=None,
         self.meta = meta.get(self.name)
 
         if len(self.meta) == 0:
-            raise FileFormatError(yml_path, logger=self.logger)
+            raise FileFormatError(yml_path, "Error getting environment name ",
+                                  logger=self.logger)
```

Comment:
There was a glitch on the way this exception was raised, adding a dummy message fix it.